app/views/hyrax/base/_metadata.html.erb

```diff
@@ -1,8 +1,4 @@
-<h2><%= t('.header') %></h2>
```

Comment:
Should we remove this from the translations then?

Reply:
Done.